MonoGame.Framework.Content.Pipeline/Serialization/Compiler/TextureCubeWriter.cs

```diff
@@ -20,6 +20,8 @@ protected internal override void Write(ContentWriter output, TextureCubeContent
             if (!level0.TryGetFormat(out format))
                 throw new Exception("Couldn't get format for TextureCubeContent.");
 
+            format = value.GetActualFormat(format);
```

Comment:
> if it's possible for an OpenGL driver to support framebuffer sRGB but not texture sRGB 

I'm not an expert... but I highly doubt it.  If the sampler has the ability to decode gamma, I would think it would be able to encode gamma.